src/app/src/pages/eval/components/ResultsTable.tsx

```diff
@@ -300,7 +313,7 @@ function ResultsTable({
   const numGoodTests = React.useMemo(
     () =>
       head.prompts.map((_, idx) =>
-        body.reduce((acc, row) => acc + (row.outputs[idx].pass ? 1 : 0), 0),
+        body.reduce((acc, row) => acc + (row.outputs[idx]?.pass ? 1 : 0), 0),
```

Comment:
This could be null because results get created as they happen now and may not all come in at once, so a test for prompt 2 could be done before prompt 1.